distrobox-create

```diff
@@ -308,10 +318,14 @@ fi
 # First, check if the image exists in the host.
 # If not prompt to download it.
 if ! ${container_manager} inspect --type image "${container_image}" >/dev/null 2>&1; then
-	# Prompt to download it.
-	printf >&2 "Image not found.\n"
-	printf >&2 "Do you want to pull the image now? [y/n] "
-	read -r response
+	if [ "${non_interactive}" == "0" ]; then
```

Comment:
We should use a single `=`, as `==` is not defined in standard POSIX sh

` 	if [ "${non_interactive}" = "0" ]; then`